spec/features/user_edits_bathroom_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+# Acceptance Criteria
+# [x] I can only see the edit button if I own the bathroom
+# [x] There is an edit button on the show page
+# [x] I can click the button to edit my bathroom
+# [x] I am linked to a page to edit my selected bathroom
+# [x] I am notified when I edit my bathroom
+
+feature "As an authenticated user
+I want to edit a bathroom
+so that bathroom has relevant information" do
+  scenario 'User edits a bathroom' do
+    @user = FactoryGirl.create(:user)
+    @bathroom = FactoryGirl.create(:bathroom, user: @user)
```

Comment:
Oh jeez can't believe I didn't notice this before, but none of these should be instance variables!! Go through all your tests and make these local variables, it's important to use the proper scope